wallet-api/src/Wallet/UTXO/Index.hs

```diff
@@ -172,10 +172,11 @@ checkMatch v = \case
             let v' = ValidationData
                     $ lifted
                     $ v { pendingTxOwnHash = Runtime.plcValidatorDigest (UTXO.getAddress a) }
+                (logOut, success) = UTXO.runScript v' vl r d
             in
-                if UTXO.runScript v' vl r d
+                if success
                 then pure ()
```

Comment:
Yeah, it's a little awkward because we don't currently return anything in the success case, so it wasn't totally clear to me how to wire that through in the most appropriate way, whereas sticking it into the validation errors is obvious.

Reply:
I'll look into it when this is merged